src/CoinSolve.cpp

```diff
@@ -192,7 +192,9 @@ void formInputQueue(std::deque<std::string> &inputQueue,
       }
    }
 }
-
+// Save a copy of input for unit testing
+// Could also be used for friendly error messages?
+std::deque<std::string> saveInputQueue;
```

Comment:
@jjhforrest Putting this here, but using it in unitTestClp.cpp doesn't work well, because unitTestClp.cpp is part of libCbc, but CoinSolve is only in the cbc executable. So with this change, I cannot use libCbc anymore:
```
/usr/bin/ld: libCbc.so: undefined reference to `saveInputQueue[abi:cxx11]'
```

Reply:
This also re-introduces the thing I was trying to eliminate: passing information through global variables instead of transparently through function arguments. Global variables are nothing but trouble. They make the program flow difficult to follow, cause problems with thread safety, etc. A big part of the refactoring was to get rid of them and it was not easy. I know we're just trying to now quickly get everything back into working order, but maybe we could at least avoid re-introducing globals. 

The input queue is passed unmodified into CbcMain1 and the unit test is called from there, so if we really need to, we can save a copy once in CbcMain1 and then pass it into the unit test as an argument. But my idea was to just to parse the command-line once, capture the information by setting parameters, etc. and then retrieve this information as needed.